build/deploy/grpc-backend.libsonnet

```diff
@@ -6,7 +6,7 @@ local volumes = import 'volumes.libsonnet';
     service: base.Service(metadata, 'grpc-backend') {
       app:: 'grpc-backend',
       port:: metadata.backend.port,
-      enable_monitoring:: true,
+      enable_monitoring:: false,
```

Comment:
you can just omit these since it defaults to false in the base

Reply:
I really prefer making it explicitly clear that monitoring is off, an when the time comes when we want to turn it on we know where it goes, more importantly the user of this repo will know by just searching "monitoring"